src/components/__tests__/TechIconList.js

```diff
@@ -13,7 +11,13 @@ const DEFAULT_PROPS = {
   ],
   flow: 'horizontal',
 };
-
+const STYLE_PROPS = {
+  icons: [
+    {name:'android', width:'18px', height:'78px',
+      style:{padding:'50px',}
```

Comment:
Please indent this better, or move style upwards o move `name` down along with width and height

Reply:
Done. Thanks.